webpack/assets/javascripts/react_app/routes/common/EmptyPage/EmptyPage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from 'patternfly-react';
+import { translate as __ } from '../../../common/I18n';
+import DefaultEmptyState from '../../../components/common/EmptyState';
+import './emptypage.scss';
+
+const EmptyPage = ({ type, text, isLoading }) => (
+  <React.Fragment>
+    {isLoading ? (
+      <div id="loading-page">
```

Comment:
can you use the LoadingState component instead?

Reply:
we can't customize `loadingState` with different skeletons like we talked about, also the component only takes care of the loading, what about empty results or a network error?
I wrote `EmptyPage` that shows a custom loading component and empty/error result if necessary.

Reply:
In my mind, they are not connected, e.g. everything that has a loading timeout, should be wrapped in a LoadingState component, what you show, is irrelevant to the loading component ? 

not sure I understand why you went this path of recreating parts of it?

Reply:
the `loadingState` can't be customized, it always shows a spinner, we want to customize the loading view per view. I tried to create a component which takes care of the loading and also of the empty/error states called EmptyPage, but we can also separate them.